plugin/pkg/shoot/validator/admission.go

```diff
@@ -471,6 +470,32 @@ func (v *ValidateShoot) Admit(a admission.Attributes, o admission.ObjectInterfac
 		allErrs = validateAlicloud(validationContext)
 	}
 
+	for idx, worker := range shoot.Spec.Provider.Workers {
+		if shoot.DeletionTimestamp == nil && worker.Machine.Image == nil {
+			shoot.Spec.Provider.Workers[idx].Machine.Image = image
+		}
+	}
+
+	if seed != nil {
+		if shoot.Spec.Networking.Pods == nil {
+			if seed.Spec.Networks.ShootDefaults != nil {
+				shoot.Spec.Networking.Pods = seed.Spec.Networks.ShootDefaults.Pods
+			} else {
+				allErrs = append(allErrs, field.Required(field.NewPath("spec", "networking", "pods"), "pods is required"))
```

Comment:
When I try to apply a v1beta1 Shoot I receive:
```
"crazy-botany" is forbidden: [spec.provider.networking.services: Required value: services is required spec.provider.networking.pods: Required value: pods is required]
```

Reply:
Yes, that's correct. If the seed doesn't have any defaults for pod/service networks then it is required that the shoot specifies them. Where should the values come from instead?